docs/root/configuration/http/http_filters/router_filter.rst

```diff
@@ -261,6 +255,25 @@ in flight.
 
 The value of the header should be "true" or "false", and is ignored if invalid.
 
+.. _config_http_filters_router_x-envoy-decorator-operation:
+
+x-envoy-decorator-operation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Setting this header on egress requests, its value will override any locally defined
```

Comment:
i rephrased. this to start the paragraph the same way as the other header, but this one is a bit different to the others because it describes two different behavior depending on whether its a request/response. 

It might be nicer to have this doc speak about downstream/upstream vs egress/ingress since I don't think the router actually cares about the ingress/egress distinction (beyond some trace tags)

Reply:
Agreed to you want to just fix that?